css-layout-classes/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Layout Classes</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <h1>Choose your character!</h1>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-two-third">
+          <img class="character" src="images/link.png" alt="Link" width="100%">
+        </div>
+        <div class="column-one-third">
+          <h2>Super Smash Bros. Ultimate</h2>
+          <p>
+            Super Smash Bros. Ultimate received universal acclaim from both
+            critics and players, with some critics calling it the best installment
+            in the series. It received praise for its large amount of content and
+            fine-tuning of existing Smash gameplay elements, although its online
+            mode was widely criticized.
+          </p>
+        </div>
+        <div class="column-two-third">
+          <img class="character" src="images/yoshi.png" alt="Yoshi" width="100%">
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="column-two-third">
+          <img class="character" src="images/kirby.png" alt="Kirby" width="100%">
+        </div>
+        <div class="column-two-third">
+          <img class="character" src="images/fox.png" alt="Fox" width="100%">
+        </div>
+        <div class="column-two-third">
+          <img class="character" src="images/pikachu.png" alt="Pikachu" width="100%">
+        </div>
+        <div class="column-two-third">
+          <img class="character" src="images/ness.png" alt="Ness" width="100%">
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="column-one-third">
```

Comment:
the text should be `column-two-thirds` since it is supposed to take up 2/3 of the total row